src/components/Textfields/TextFieldBase/index.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+    TextField as MaterialTextField,
+    BaseTextFieldProps as MaterialTextFieldProps,
+    makeStyles,
+    Theme,
+    Box,
+} from "@material-ui/core";
+
+export interface TexfieldBaseProps extends MaterialTextFieldProps {
+    label: string;
+    name?: string;
+    type?: string;
+    disabled?: boolean;
+    autoFocus?: boolean;
+    fullWidth?: boolean;
+    errorMessage?: string;
+    inputRef?: React.RefObject<HTMLInputElement>;
```

Comment:
Is there a reason for explicitly declare this?

Reply:
Because when TextfieldBaseProps extends BaseTextFieldProps, the storybook only show 3 props (ref, focused, hiddenLabel), so I declare more props to user can see more props in storybook. And I also want to custom error and error message by only one props is errorMessage.